proto/kvrpcpb.proto

```diff
@@ -766,6 +766,13 @@ enum IsolationLevel {
     RC = 1; // RC = read committed
 }
 
+//Operation allowed level during each TiKV disk full threshold.
+enum AllowedLevel {
+    Allowed_Normal = 0; // Normal is the default value, meaning all operations are allowed when disk status is normal.
```

Comment:
It's better to be `Normal`, `AllowedOnAlmostFull` and `AllowedOnAlreadyFull`.

Reply:
changed